tests/cross/src/test/scala/tests/pc/CompletionCaseSuite.scala

```diff
@@ -291,6 +291,10 @@ class CompletionCaseSuite extends BaseCompletionSuite {
     """|case None => scala
        |case Some(value) => scala
        |""".stripMargin,
+    compat = Map("3" -> """|case None => scala
```

Comment:
Btw. could you follow up with an issue for the same feature in Scala 2?

Reply:
support for this in Scala 2 would be awesome!